aptibleapi/.openapi-generator/FILES

```diff
@@ -194,9 +217,17 @@ docs/ListEphemeralContainersForEphemeralSession200ResponseLinks.md
 docs/ListEphemeralSessionsForApp200Response.md
 docs/ListEphemeralSessionsForApp200ResponseEmbedded.md
 docs/ListEphemeralSessionsForApp200ResponseLinks.md
+docs/ListExternalAwsAccounts200Response.md
+docs/ListExternalAwsAccounts200ResponseEmbedded.md
+docs/ListExternalAwsDatabaseCredentialsForExternalAwsResource200Response.md
+docs/ListExternalAwsDatabaseCredentialsForExternalAwsResource200ResponseEmbedded.md
+docs/ListExternalAwsDatabaseCredentialsForExternalAwsResource200ResponseLinks.md
+docs/ListExternalAwsResources200Response.md
+docs/ListExternalAwsResourcesForExternalAwsAccount200Response.md
+docs/ListExternalAwsResourcesForExternalAwsAccount200ResponseEmbedded.md
+docs/ListExternalAwsResourcesForExternalAwsAccount200ResponseLinks.md
 docs/ListImagesForApp200Response.md
 docs/ListImagesForApp200ResponseEmbedded.md
-docs/ListImagesForApp200ResponseLinks.md
```

Comment:
This seems unrelated to your intentional changes, do we know why it was removed?